example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -320,7 +320,7 @@
 				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = W46X4FW8UP;
 				HEADER_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = example/Info.plist;
-				INFOPLIST_KEY_CFBundleDisplayName = "Dev RN PhylloSampleApp";
+				INFOPLIST_KEY_CFBundleDisplayName = "Dev RN SampleApp";
```

Comment:
Any reason to change this?

Reply:
@akshaybhalotia no reason , give me one specific example name for this . itsn't able to show full name when we install example app in iphone so make it short .